backend/src/test/java/gov/cdc/usds/simplereport/db/model/PersonSerializationTest.java

```diff
@@ -46,11 +51,16 @@ public void serialize_withOrgAndFacility_noOrgOrFacility() throws IOException {
         final int BIRTH_MONTH = 3;
         final int BIRTH_DAY = 31;
         Organization fakeOrg = new Organization("ABC", "123");
-        Person p = new Person(fakeOrg,
-                null, "John", "Jacob", "Jingleheimer-Jones", "Jr.", LocalDate.of(BIRTH_YEAR, BIRTH_MONTH, BIRTH_DAY),
-                null, "1234556",
-                null, "a@b.c", "marathon", "generic", "Male-ish", true, false);
-        p.setFacility(new Facility(fakeOrg, "Nice Place", "YOUGOTHERE", null));
+        Person p = new Person(fakeOrg, null, "John", "Jacob", "Jingleheimer-Jones", "Jr.",
+                LocalDate.of(BIRTH_YEAR, BIRTH_MONTH, BIRTH_DAY), null, "1234556", null, "a@b.c", "marathon", "generic",
+                "Male-ish", true, false);
+        Provider mccoy = new Provider("Doc", "", "", "", "NCC1701", null, "(1) (111) 2222222");
+        List<DeviceType> configuredDevices = new ArrayList<>();
+        DeviceType bill = new DeviceType("Bill", "Weasleys", "1", "12345-6");
+        configuredDevices.add(bill);
+        StreetAddress addy = new StreetAddress(Collections.singletonList("Moon Base"), "Luna City", "THE MOON", "", "");
+        p.setFacility(new Facility(fakeOrg, "Nice Place", "YOUGOTHERE", addy, "555-867-5309", "facility@test.com",
```

Comment:
That was probably annoying mildly annoying to do, so thank you for putting in the extra effort to get rid of that constructor!